### DISCUSSION TITLE
How to fill the grid row if some div is missing?

### DISCUSSION THREAD

**Discussion by frederikhors** [tailwindlabs/tailwindcss#10459] [Help]
I'm using tailwindcss and this html:

REPL: https://play.tailwindcss.com/L8McFjGBVC

```html
<div class="grid grid-cols-12">
  <div class="col-span-4 bg-red-300">1</div>
  <div class="col-span-4 bg-green-300" v-if="showSecondDiv">2</div>
  <div class="col-span-4 bg-blue-300">3</div>
</div>
```

The second div can sometimes be missing.

I would like the third div to fill the row (together with the first one), as if it had class "col-span-8".

How can I fix?

**Comment by wongjn** [comment]
You could apply `[&:nth-child(2)]:col-span-8`: https://play.tailwindcss.com/WdHbLyNsUX
